releases/8.1/languages/en.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+return [
+    'common_header' => 'PHP 8.1 is a major update of the PHP language. enums, readonly properties, first-class callable syntax, fibers, intersection types, performance improvements and more.',
+    'main_title' => 'Released!',
+    'main_subtitle' => 'PHP 8.1 is a major update of the PHP language.<br class="display-none-md"> It contains many new features and optimizations including enums, readonly&nbsp;properties, first-class&nbsp;callable&nbsp;syntax, fibers, intersection&nbsp;types, performance improvements and more.',
+    'upgrade_now' => 'Upgrade to PHP 8.1 now!',
+    'documentation' => 'Doc',
+
+    'enumerations_title' => 'Enumerations',
+    'enumerations_content' => 'Use enum instead of a set of constants and get validation out of the box.',
+
+    'readonly_properties_title' => 'Readonly Properties',
+    'readonly_properties_content' => '<p>Readonly properties cannot be changed after initialization, i.e. after a value is assigned to them.<br>They are a great way to model value objects and data-transfer objects.</p>',
+
+    'first_class_callable_syntax_title' => 'First-class Callable Syntax',
+    'first_class_callable_syntax_content' => '<p>It is now possible to get a reference to any function – this is called first-class callable syntax.</p>',
+
+    'new_in_initializers_title' => 'New in initializers',
+    'new_in_initializers_content' => '<p>Objects can now be used as default parameter values, static variables, and global constants, as well as in attribute arguments.</p>
+            <p>This effectively makes it possible to use <strong>nested attributes</strong>.</p>',
+
+    'pure_intersection_types_title' => 'Pure Intersection Types',
+    'pure_intersection_types_content' => '<p>Use intersection types on any type declarations when you need to require a value to satisfy multiple type constraints at the same time.</p>
```

Comment:
```suggestion
    'pure_intersection_types_content' => '<p>Use intersection types when you need to require a value to satisfy multiple type constraints at the same time.</p>
```
This read a bit awkwardly to me, and don't think it needs to be spelled out.